models/get.models.js

```diff
@@ -8,3 +8,15 @@ exports.getSlugAndDescription = () => {
         return topics.rows;
     }) 
 };
+
+
+exports.retreiveArticles = (article_id) => {
```

Comment:
Can we please change the names of the functions used here as we are only retrieving a singular article, the next ticket is for multiple articles, so you are better to use those function names for that ticket. This one should Ideally be retrieveArticle and getArticle